src/Route/RouteHelper.php

```diff
@@ -90,37 +64,26 @@ public function appendQueryParam(UriInterface $uri, string $name, $value): UriIn
         );
 
         $uri = $uri->withQuery($query);
+
         return $uri;
```

Comment:
```suggestion
        return $uri->withQuery($query);
```
And remove `$uri = $uri->withQuery($query);` above;